cling/exception/CMakeLists.txt

```diff
@@ -3,9 +3,9 @@ if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
                     MACRO nullderef.C
                     PASSRC 1
                     LABELS roottest regression cling)
+else()
+  ROOTTEST_ADD_TEST(nullderef-e
+                    MACRO nullderef_exception.C
+                    PASSRC 1
+                    LABELS roottest regression cling)
 endif()
-
-ROOTTEST_ADD_TEST(nullderef-e
```

Comment:
I do not understand why this change is needed. The commit log does not explain either.